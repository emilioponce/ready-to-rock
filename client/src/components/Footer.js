import React from 'react'

import styles from './Footer.module.css'

const Footer = () => (
  <div className={styles.footer}>
    <div className={styles.text}>
      Ready to rock Barcelona by now is only a proof of concept for testing
      purposes..
    </div>
  </div>
)

export default Footer
