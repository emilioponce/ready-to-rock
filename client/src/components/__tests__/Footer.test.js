import React from 'react'
import ReactDOM from 'react-dom'
import Footer from '../Footer'
import { shallow } from 'enzyme'

const component = <Footer />

describe('Component Footer', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(component, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('snapshot renders correctly', () => {
    const wrapper = shallow(component)
    expect(wrapper).toMatchSnapshot()
  })
})
