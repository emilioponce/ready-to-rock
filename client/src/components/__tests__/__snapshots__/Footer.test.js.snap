// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Footer snapshot renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="text"
  >
    Ready to rock Barcelona by now is only a proof of concept for testing purposes..
  </div>
</div>
`;
